odbc/src/api/functions.rs

```diff
@@ -409,13 +411,9 @@ pub unsafe extern "C" fn SQLColAttributeW(
             };
             let numeric_col_attr = |f: &dyn Fn(&MongoColMetadata) -> Len| {
                 let mongo_handle = MongoHandleRef::from(statement_handle);
-                let stmt = must_be_valid!((*mongo_handle).as_statement());
+                let stmt = (*mongo_handle).as_statement().unwrap();
                 {
                     let stmt_contents = stmt.read().unwrap();
-                    if stmt_contents.mongo_statement.is_none() {
-                        *numeric_attribute_ptr = 0 as Len;
-                        return SqlReturn::SUCCESS;
-                    }
                     let col_metadata = stmt_contents
```

Comment:
I don't know what unnamed and updatable mean.

No we are not handling IRD or ARD. _id is part of the data for us, and that would be the closest equivalent.

Reply:
> I don't know what unnamed and updatable mean.

⬆️ This is what I am hinting at. We don't really know what we are returning here.

We return 0, but 0 has a meaning. I'm just thinking if we have a comment stating what we are returning, it will feel less like magic and also helps making sure that we are returning the correct value.
See https://learn.microsoft.com/en-us/sql/odbc/reference/syntax/sqlcolattribute-function for the attribute description and possible values. 
For SQL_DESC_NAME, 0=SQL_NAMED, see https://github.com/microsoft/ODBC-Specification/blob/b7ef71fba508ed010cd979428efae3091b732d75/Windows/inc/sql.h#L296
For SQL_DESC_UPDATABLE, 0=SQL_ATTR_READONLY, see https://github.com/microsoft/ODBC-Specification/blob/b7ef71fba508ed010cd979428efae3091b732d75/Windows/inc/sqlext.h#L707.
Also, if 

Good to know that we don't need IRD. I would have to start looking into again otherwise because I forgot everything about it, except that it can be a pain.

Reply:
Oh, we aren't returning 0 anymore. I was returning 0 before because it was empty. But I think the best behavior (even though the behavior is actually undefined), is to store nothing to the pointers.